eos-extra-resize

```diff
@@ -32,9 +32,16 @@ if [ -z "$extra_disk" ]; then
     exit 0
 fi
 
+pt_label=$(blkid -o value -s PTTYPE $extra_disk)
+
+if [ "$pt_label" = "gpt" ]; then
+  marker=$(sfdisk --force --part-attrs $extra_disk 1)
+else
+  marker=$(sfdisk --force --part-type $extra_disk 4)
```

Comment:
Am I wrong, or are you reading the wrong partition in the MBR case? There aren't 4 partitions in the extra disk.